src/Makefile

```diff
@@ -1,13 +1,19 @@
+##
+# Formal Ledger Specification
+#
+# @file
+# @version 0.1
+
 # Constants
 AGDA?=agda
 AGDA_RUN=$(AGDA) --transliterate
 OUT_DIR?=../dist
-LATEX?=latexmk -xelatex -shell-escape -halt-on-error
+LATEX?=latexmk -xelatex -shell-escape -halt-on-error -gg
```

Comment:
I'm not sure we want this option here. According to the manual it forces recompilation of everything.
```
-gg "Super go mode" or "clean make": clean out generated files as if -C had been given, and then do a
regular make."
```

Reply:
Yeah, I added that when I got frustrated by the "already up-to-date" error, when, in fact, the document was not up-to-date.  I'll remove it.